src/view.rs

```diff
@@ -127,12 +127,21 @@ impl<'a> View<'a> {
       if let Some(ref hint) = mat.hint {
         let extra_position = if self.position == "left" {
```

Comment:
Maybe a `match` will look much better here

Reply:
Ok cleaned that up and you were right.
Sorry Rust is not my primary language and this _may_ in fact be the first time I've tinkered with it. ;)